pyvista/core/pointset.py

```diff
@@ -818,6 +827,79 @@ def faces(self, faces):
             # TODO: faster to mutate in-place if array is same size?
             self.SetPolys(CellArray(faces))
 
+    @property
+    def regular_faces(self) -> np.ndarray:
+        """Return a face array of point indices when all faces have the same size.
+
+        Returns
+        -------
+        numpy.ndarray
+            Array of face indices with shape (n_faces, face_size).
+
+        See Also
+        --------
+        pyvista.PolyData.faces
+
+        Notes
+        -----
+        This property does not validate that the mesh's faces are all
+        actually the same size. If they're not, this property may either
+        raise a `ValueError` or silently return an incorrect array.
+
+        Examples
+        --------
+        Get the face array of a tetrahedron as a 4x3 array
+
+        >>> import pyvista as pv
+        >>> tetra = pv.Tetrahedron()
+        >>> tetra.regular_faces
+        array([[0, 1, 2],
+               [1, 3, 2],
+               [0, 2, 3],
+               [0, 3, 1]])
+
+        """
+        return _get_regular_cells(self.GetPolys())
+
+    @regular_faces.setter
+    def regular_faces(self, faces: Union[List[List[int]], np.ndarray]):
```

Comment:
Forgot `regular_lines`, which could be more useful than `regular_strips`.

Reply:
I haven't really used strips much. Do you mean that `strips=[4, a, b, c, d]` implies `regular_strips=[[a, b, c], [b, c, d]]` ? That might be less useful just because you lose the benefit of a cheap view into the underlying connectivity array.

Agreed on `regular_lines`, which I think I would use especially for `mesh.extract_all_edges().regular_lines` to return `n_edges x 2` array.

Reply:
strips would actually be `strips=[4, a, b, c, d]` and `regular_strips=[[a, b, c, d]]`.  More illustrative would be `strips=[4, a, b, c, d, 4, e, f, g, h]` and `regular_strips=[[a, b, c, d]. [e, f, g, h]`.  triangle strips are a weird cell type as they have multiple triangles defined in one cell.  It is sort of similar to `PolyLine` and `PolyVertex`

Reply:
Oh got it. Yeah that would be an easy extension; I've it for a followup PR once this gets nailed down